decidim-core/spec/commands/decidim/unsubscribe_notifications_settings_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  describe UnsubscribeNotificationsSettings do
+    let(:command) { described_class.new(user) }
+    let(:user) { create(:user, newsletter_notifications: true) }
+    let(:newsletter_notifications) { true }
+
+    context "when newsletter notifications are true" do
+      let(:newsletter_notifications) { true }
+
+      it "Unsubscribe user" do
```

Comment:
These tests are really brittle. We should be setting expectations on what the command side-effects are.